app/code/community/Ebizmarts/MailChimp/etc/system.xml

```diff
@@ -217,7 +223,8 @@
                             <show_in_default>1</show_in_default>
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
-                            <comment><![CDATA[Click this button to create the webhook manually, it will only work if the webhook doesn't already exist.]]></comment>
+                            <comment>
```

Comment:
This blank lines after the start tag cause failure.

Reply:
Solved.